### DISCUSSION TITLE
Encountering Database Initialization error during installation

### DISCUSSION THREAD

**Discussion by samsonyuwono** [requarks/wiki#2886] [Help / Questions]
When I run `node server`, I keep running into the error below and am not able to complete the installation.

![Screen Shot 2021-01-05 at 4 33 15 PM](https://user-images.githubusercontent.com/25911721/103701288-bf0bff80-4f73-11eb-8d5b-864b7136803b.png)

What's going on here? Are there changes I have to make to the files for which I'm not aware of.

**Comment by NGPixel** [comment]
What database version are you using?
